### DISCUSSION TITLE
[Exercise-13]: Create trigger TRG_AUDIT_FDS_ITEMS

### DISCUSSION THREAD

**Discussion by acrlakshman** [exaslate-courses/oracle-sql-plsql-b3#15] [General]
* Create trigger to log data into AUDIT_FDS_ITEMS table from exercise-12 (https://github.com/exaslate-courses/oracle-sql-plsql-b3/discussions/14).

Ref.: https://www.youtube.com/watch?v=AoIJ20d803A&t=384s

**Comment by srideviadapa0511** [comment]
![trigger](https://github.com/exaslate-courses/oracle-sql-plsql-b3/assets/145452411/611b61dd-bd2b-4c17-9af0-6da6eef32e32)

**Reply by acrlakshman** [reply]
@srideviadapa0511 this looks good, now can you add some code in `BEGIN` so that when trigger is called, the above one will INSERT new record into `AUDIT_FDS_ITEMS` table.

**Comment by srideviadapa0511** [comment]
![updated trigger](https://github.com/exaslate-courses/oracle-sql-plsql-b3/assets/145452411/e610a4c0-9b83-4bf1-b4ef-e7e95150e7a4)
